geometry_msgs/Point pick_pose
geometry_msgs/Point place_pose
string hand
---
bool done
